File: Assignment_23/program2.c
// Accept character from user. If character is small display its corresponding capital character,  
// and if it small then display its corresponding capital. In other cases display as it is.

///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Accept Character from user and if character small then display its ccorresponding capital character
//                and vice versa
// Autor       :  Ankita Anil Patil
// Date        :  31/05/2025
///////////////////////////////////////////////////////////////////////////////



/*
    Input : Q
    OUtput: q

    Input : m
    OUtput: M

    Input : 4
    OUtput: 4

    Input : %
    OUtput: %
*/

#include<stdio.h>

void Display(char ch)
{
    char cCap,cSmall = '\0';

    cCap = 'A';
    cSmall = 'a';

    if((ch >= 'A') && (ch <= 'Z'))
    {
        for(cCap = 'A'; ch <= 'Z'; cCap++)
        {
            
            if(cCap == ch)
            {
                break;
            }
            cSmall++;

        }

        printf("%c\n",cSmall);

        

    }
    else if((ch >= 'a') && (ch <= 'z'))
    {
        for(cSmall = 'a'; ch <= 'z'; cSmall++)
        {
            
            if(cSmall == ch)
            {
                break;
            }
            cCap++;

        }

        printf("%c\n",cCap);
    }

    else
    {
        printf("%c\n",ch);
    }
    
}

int main()
{
    char cValue = '\0';

    printf("Enter Character  \n");
    scanf("%c",&cValue);

    Display(cValue);

    return 0;
}

File: Assignment_53/program2.java
// Write a java program which accept two two arrays from user and display minimum element of each array. 
/*
    Input :  2 9 7 5 2 3 
             9 3 5 5 
    Output : 2 3
*/ 

 ///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.java
// Description :  Accept two arrays from user and display minimum element of each array.          
// Autor       :  Ankita Anil Patil
// Date        :  13/09/2025
///////////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray 
{ 
        public void MinArray(int Arr1[], int Arr2[]) 
        {   
            int i = 0;
            int iMin1 = 0, iMin2 = 0;

            iMin1 = Arr1[0];
            for(i = 0; i < Arr1.length; i++)
            {
                if(Arr1[i] < iMin1)
                {
                    iMin1 = Arr1[i];
                }
            }

           
             iMin2 = Arr2[0];
            for(i = 0; i < Arr2.length; i++)
            {
                if(Arr2[i] < iMin2)
                {
                    iMin2 = Arr2[i];
                }
            }

            System.out.println("Minimum element from array 1 is : "+ iMin1);
            System.out.println("Minimum element from array 2 is : "+ iMin2);

        } 
} 
public class program2 
    
{   
    public static void main(String[] args) {
        

    Scanner sobj = new Scanner(System.in);


    System.out.println("Array 1: ");
    System.out.println("Enter size of array");
    int iSize = sobj.nextInt();


    int[] Arr1 = new int[iSize];
    System.out.println("Enter array 1 elements : ");

    int i = 0;
    for(i = 0; i < Arr1.length; i++)
    { 
       Arr1[i] = sobj.nextInt();
    }

    System.out.println("Array 2: ");
    System.out.println("Enter size of array");
    int iSize2 = sobj.nextInt();


    int[] Arr2 = new int[iSize2];
    System.out.println("Enter array  2 elements : ");

    for(i = 0; i < Arr2.length; i++)
    {
        Arr2[i] = sobj.nextInt();
    }
    
    MyArray mobj = new MyArray();
    mobj.MinArray(Arr1, Arr2);

    
    
}
}
